static/js/publisher/release/components/progressiveBar.js

```diff
@@ -1,7 +1,40 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-class ProgressiveBar extends React.Component {
+const ProgressiveBar = ({ percentage, targetPercentage, readonly }) => (
```

Comment:
https://youtu.be/__X6cYJDXOg?t=11

Reply:
I don't think the people that uploaded that video know what "High res" means.